addons/skin.estuary/xml/Variables.xml

```diff
@@ -307,11 +307,12 @@
 		<value>$INFO[ListItem.PictureDateTime]</value>
 	</variable>
 	<variable name="NowPlayingBreadcrumbsVar">
-  		<value condition="MusicPlayer.Content(livetv) + Player.HasAudio">$INFO[MusicPlayer.Title]</value> 
+		<value condition="MusicPlayer.Content(livetv) + Player.HasAudio">$INFO[MusicPlayer.Artist,, - ]$INFO[MusicPlayer.Title]</value>
```

Comment:
Honestly, I don't even remember doing that!  There is a screenshot of it above but I can revert it if you would prefer.

Reply:
I think this change is unrelated to this PR and should be kept out here.

Reply:
Ah, remembered!!  A v1 stream will be in the format `artist - title` and that goes into `MusicPlayer.Title` so automatically shows both artist and title in the one label.  For v2 streams, the artist and title are separated out so without the change you wouldn't see the artist name, just the title of the song.  `MusicPlayer.Artist` is empty for v1 streams so this change makes the display consistent for both versions.

Reply:
Don't kill me, but shouldn't v1 be fixed then not to include artist in the title guiinfo label?

Reply:
Both Artist info and Track Title should always be displayed, a Track Title alone can't identify a song without the Artist.

Reply:
Looking Music Videos the OSDSubLabelVar
is used for Artist/Album, however this can't be used as Channel name is used in OSDSubLabelVar for PVR. See https://github.com/xbmc/xbmc/blob/master/addons/skin.estuary/xml/Variables.xml#L330